tests/unit/lib/PrinterHelpers.spec.js

```diff
@@ -7,39 +7,39 @@ describe('PrinterHelpers', () => {
     it('should return an empty array if no printers are provided', () => {
       const result = filterPrintersByEnvironment({
         printers: [],
-        environment: 'production',
+        hideNonProductionPrinters: false,
```

Comment:
This was less difficult for me, but while I'm making comparisons, perhaps this should be inverted as well.  `onlyShowProductionPrinters` perhaps?